src/server/page_server.cpp

```diff
@@ -242,8 +242,11 @@ page_server::set_active_tran_server_connection (cubcomm::channel &&chn)
   assert (is_page_server ());
 
   chn.set_channel_name ("ATS_PS_comm");
-  er_log_debug (ARG_FILE_LINE, "Active transaction server connected to this page server. Channel id: %s.\n",
-		chn.get_channel_id ().c_str ());
+  if (chn.get_port() != -1)
+    {
+      er_log_debug (ARG_FILE_LINE, "Active transaction server connected to this page server. Channel id: %s.\n",
```

Comment:
The issue comment says to not include port in `chn.get_channel_id ()` if its value is -1. These message should be logged nonetheless.